refresh/config.go

```diff
@@ -24,7 +26,13 @@ type Configuration struct {
 }
 
 func (c *Configuration) FullBuildPath() string {
-	return path.Join(c.BuildPath, c.BinaryName)
+	path := path.Join(c.BuildPath, c.BinaryName)
+	if runtime.GOOS == "windows" {
+		if !strings.HasSuffix(path, ".exe") {
```

Comment:
this may not be a big issue, but do note that HasSuffix is case sensitive (e.g. resulting in app.EXE.exe)